packages/core/src/GenericBanner/GenericBanner.stories.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { GenericBanner } from '.'
+import { Warning, Rowing, Tea, Emoticon } from 'pcln-icons'
+import { Text } from '../Text'
+import styled from 'styled-components'
+import { getPaletteColor } from '../utils'
+
+export default {
+  title: 'GenericBanner',
+  component: GenericBanner,
+}
+
+const UnderlinedLinkText = styled(Text)`
+  z-index: 2;
+  text-decoration: underline;
+  &:hover {
+    color: ${(props) => getPaletteColor(props.hoverColor)};
+  }
+`
+
+export const basicBannerAllInline = () => (
+  <GenericBanner
+    p={2}
+    heading={
+      <Text.span fontWeight='bold' textColor='primary.base'>
+        COVID-19&nbsp;&nbsp;
+      </Text.span>
+    }
+    text={
+      <Text.span textColor='primary.base'>
+        Update: Your travel may be impacted. Please review this hotels important
+        info.
+      </Text.span>
+    }
+    iconLeft={<Warning />}
+    color='caution.light'
+  />
+)
+export const basicBannerAllInlineRightIcon = () => (
+  <GenericBanner
+    p={2}
+    heading={
+      <Text.span fontWeight='bold' color='secondary'>
+        This is a Heading&nbsp;&nbsp;
```

Comment:
n-b it looks like your editor might have converted some unneeded whitespace

Reply:
Ah I see what those were for; replaced with `mr` to make it a bit cleaner.

Reply:
Yes exactly! Sorry left this comment open so I could respond saying that and then forgot to actually write that comment 😆